core/controllers/base.py

```diff
@@ -358,14 +357,18 @@ def validate_and_normalize_args(self):
         """
         handler_class_name = self.__class__.__name__
         request_method = self.request.environ['REQUEST_METHOD']
+        if request_method == 'HEAD':
+            request_method = 'GET'
```

Comment:
Explain why we do this with a code comment.

Reply:
done